### DISCUSSION TITLE
How do get Pair Extraordinaire badge?

### DISCUSSION THREAD

**Discussion by alpine-storm** [juddydev/react-3D#7] [Q&A] [answered]
How do get Pair Extraordinaire badge?

**Comment by t0linixer** [comment, marked as answer]
you need to create co-authored commits on the command line(in a terminal way).

One thing to note is that the closing quotation mark(") must be written
after the co-author's email, and the email must be enclosed in <>

(more details )
https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

**Reply by alpine-storm** [reply]
thank you

**Comment by SUDEEP-M-SHETTY** [comment]
Go to this repository
https://github.com/Prakash4844/Get-Pair-Extraordinaire
Follow the steps provided in the repository and you'll have the badge in no time. 
This is the easiest way you can obtain it.

**Reply by alpine-storm** [reply]
thank you for your help

**Reply by SUDEEP-M-SHETTY** [reply]
Hope this answers your question😄

**Comment by t0linixer** [comment]
Good luck!
